plugins/inputs/jolokia2/client.go

```diff
@@ -153,12 +154,12 @@ func (c *Client) read(requests []ReadRequest) ([]ReadResponse, error) {
 		return nil, err
 	}
 
-	var jresponses []jolokiaResponse
-	if err = json.Unmarshal([]byte(responseBody), &jresponses); err != nil {
-		return nil, fmt.Errorf("Error decoding JSON response: %s: %s", err, responseBody)
+	var jResponses []jolokiaResponse
+	if err = json.Unmarshal(responseBody, &jResponses); err != nil {
+		return nil, fmt.Errorf("error decoding JSON response: %s: %s", err, responseBody)
```

Comment:
I think we drop 'error' too?
```suggestion
		return nil, fmt.Errorf("decoding JSON response: %s: %s", err, responseBody)
```

Reply:
Fixed.